foe-ios/Models/Sighting.swift

```diff
@@ -55,7 +55,7 @@ class Sighting {
   }
 
   func getHabitat() -> String {
-    return self.habitat!
+    return (self.habitat != nil) ? self.habitat! : "";
```

Comment:
Is habitat ever nil? If so, should we make this a required field in the submission (i.e. details controller)?

Reply:
same as weather; i use this comparison to evaluate whether the user filled these fields out for submission